### DISCUSSION TITLE
Lower FPS = Better Accuracy?

### DISCUSSION THREAD

**Discussion by Re3der** [ppy/osu#22510] [Q&A]
I'm a mania player,and when I lower my FPS to vsync (165 FPS), my acc is better than on unlimited mode or optimal. I discovered same issue on stepmania - fullscreen windowed + 165 FPS = better acc than fullscreen. Does someone know what is wrong or why is this happening? I can also add evidence if needed.

**Comment by mk56-spn** [comment]
> I'm a mania player,and when I lower my FPS to vsync (165 FPS), my acc is better than on unlimited mode or optimal. I discovered same issue on stepmania - fullscreen windowed + 165 FPS = better acc than fullscreen. Does someone know what is wrong or why is this happening? I can also add evidence if needed.

The only reason I can think of is that when you set it higher ( say 330 or 660) your GPU can't push out enough frames to be stable at that FPS so it feels choppy and makes you lose accuracy. Although considering you mention  stepmania and that game is two decades old that seems improbable on any reasonably current device .
